README.md

```diff
@@ -204,228 +215,251 @@ It is possible to configure the `macvlan` driver to bridge your container to the
+**Default:** `1024M`
 
-Ex:
+### External MongoDB environment variables
 
-```
---env JVM_MAX_THREAD_STACK_SIZE=1280k
-```
+These variables are used to implement support for an [external MongoDB server](https://community.ubnt.com/t5/UniFi-Wireless/External-MongoDB-Server/td-p/1305297) and must all be set in order for this feature to work. Once all are set then the configuration file value for `db.mongo.local` will automatically be set to `false`.
 
-as a fix for https://community.ubnt.com/t5/UniFi-Routing-Switching/IMPORTANT-Debian-Ubuntu-users-MUST-READ-Updated-06-21/m-p/1968251#M48264
+* `DB_URI`
+Maps to `db.mongo.uri`.
 
-### `LOTSOFDEVICES`
-Enable this with `true` if you run a system with a lot of devices and/or
-with a low powered system (like a Raspberry Pi)
-This makes a few adjustments to try and improve performance: 
+* `STATDB_URI`
+Maps to `statdb.mongo.uri`.
 
-* enable unifi.G1GC.enabled
-* set unifi.xms to JVM_INIT_HEAP_SIZE
-* set unifi.xmx to JVM_MAX_HEAP_SIZE
-* enable unifi.db.nojournal
-* set unifi.dg.extraargs to --quiet
+* `DB_NAME`
+Maps to `unifi.db.name`.
 
-See [This website](https://help.ui.com/hc/en-us/articles/115005159588-UniFi-How-to-Tune-the-Network-Application-for-High-Number-of-UniFi-Devices) for an explanation 
-of some of those options.
+## Exposed Ports
 
-Default: `unset`
+This container exposes the following ports.
+A minimal Unifi Controller installation requires you
+expose the first three with the `-p ...` option.
 
-### `JVM_EXTRA_OPTS`
-Used to start the JVM with additional arguments.
+* 8080/tcp - Device command/control 
+* 8443/tcp - Web interface + API 
+* 3478/udp - STUN service 
+* 8843/tcp - HTTPS portal _(optional)_
+* 8880/tcp - HTTP portal _(optional)_
+* 6789/tcp - Speed Test (unifi5 only) _(optional)_
 
-Default: `unset`
+See [UniFi - Ports Used](https://help.ubnt.com/hc/en-us/articles/218506997-UniFi-Ports-Used) for more information.
 
-### `JVM_INIT_HEAP_SIZE`
-Set the starting size of the javascript engine for example: `1024M`
+## Run as non-root User
 
-Default: `unset`
+The default container runs Unifi Controller as root.
+The recommended `docker run...` command above starts
+Unifi Controller so the image runs as `unifi` (non-root)
+user with the uid/gid 999/999.
+You can also set your data and logs directories to be
+owned by the proper gid.
 
-### `JVM_MAX_HEAP_SIZE`
-Java Virtual Machine (JVM) allocates available memory. 
-For larger installations a larger value is recommended. For memory constrained system this value can be lowered. 
+_Note:_ When you run as a non-root user,
+you will not be able to bind to lower ports by default.
+(This would not necessary if you are using the default ports.)
+If you must do this, also pass the 
+`--sysctl net.ipv4.ip_unprivileged_port_start=0`
+option on the `docker run...` to bind to whatever port you wish.
 
-Default `1024M`
+~~## Unifi-in-Docker on Windows~~
```

Comment:
@tiesmaster says it works on windows without issue so we can probably drop this windows section. If someone reports its still an issue I can always add this section back again

Reply:
Agree